TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -397,7 +397,7 @@ static inline void reDrawProgress(uint8_t prevProgress)
 {
   uint8_t nextProgress = getPrintProgress();
 
-  if (nextProgress > prevProgress)
+  if (nextProgress != prevProgress)
```

Comment:
It surely won't fix it as progress must go upwards and as you seen in issue #2479 it stopped at 11%.